packages/prosess-foreldelse/src/components/ForeldelsePeriodeForm.tsx

```diff
@@ -125,6 +152,8 @@ const buildInitalValues = (periode: ForeldelsesresultatActivity) => ({
 });
 
 interface PureOwnProps {
+  behandlingId: number;
```

Comment:
Giddar du fikse opp i bruker av PureOwnProps her... er eg som har gjort det altso, men fint om du giddar fikse opp. Sjå til dømes fakta-adopsjon for korleis eg brukar PureOwnProps, MappedOwnProps og FaktaValues.